member/src/it/scala/MemberServerSpec.scala

```diff
@@ -33,18 +35,27 @@ class MemberServerSpec extends ServiceTestContainerSpec(8081, "member-service")
         userName = "SomeUserName",
       )
     ),
-    organizations = Seq(OrganizationId("SomeOrganization")),
+    organizationMembership = Seq(OrganizationId("SomeOrganization")),
     tenant = Some(TenantId("SomeTenant"))
   )
-  var memberId: MemberId = null
-  val memberInfoWithMobNumber = memberInfo.withContact(memberInfo.contact.get.withPhone("123-456-7890"))
```

Comment:
You've used one of my pet peeves in naming: using a phoneme that has a meaning. You've likely contracted "Mobile" to "Mob" but "Mob" has a non-relevant meaning. This adds cognitive load to the reader. I'd rather you spelled it out.

Reply:
Oops, that was a weird artifact from Shail's PRs. I copied it not knowing what it was and also had a pet peeve in that and changed another thing that had Mob, but forgot this one. I'll go ahead and fix this.

Reply:
since it was only used in one test and the cognitive load was simple, I went ahead and just put the copied memberInfo into the test and removed this val completely.